from mysql.connector import Connect

# Making connection
try:
    con = Connect(
        host ="localhost",
        port = 3307,
        user = "root"
        password = "Appu_2003",
        database = "pydb"
        )
expect Error as e